src/bundles/BuyerDashboard/components/Views/Views.js

```diff
@@ -0,0 +1,249 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import isEmpty from 'lodash/isEmpty';
+import format from 'date-fns/format';
+
+import './Views.css'
+
+
+export const DashboardBriefs = ({live, closed, draft, all}) => (
```

Comment:
That whole folder was redundant after I split out the above and I deleted it but it one of my follow up commits it came back. Will remove.